msbuild/Xamarin.iOS.Tasks.Core/Xamarin.iOS.Common.targets

```diff
@@ -1772,8 +1802,16 @@ Copyright (C) 2013-2016 Xamarin. All rights reserved.
 		</CollectFrameworks>
 	</Target>
 
+	<PropertyGroup>
+		<_BeforeCodesignNativeLibrariesForBuildDependsOn>
+			_CompileToNative
+		</_BeforeCodesignNativeLibrariesForBuildDependsOn>
+	</PropertyGroup>
+
+	<Target Name="_BeforeCodesignNativeLibrariesForBuild" Condition="'$(IsAppDistribution)' != 'True'" BeforeTargets="_CodesignNativeLibraries" DependsOnTargets="$(_BeforeCodesignNativeLibrariesForBuildDependsOn)" />
```

Comment:
It doesn't look like anything depends on `_BeforeCodesignNativeLibrariesForBuild`, is that on purpose?

Reply:
No, nothing depends on it but it will always run before `_CodesignNativeLibraries` because of the `BeforeTargets` property.